docs/examples/offloading-multiple-pods.md

```diff
@@ -0,0 +1,251 @@
+liqoctl peer in-band --remote-kubeconfig "$KUBECONFIG_DESTINATION1"
+liqoctl peer in-band --remote-kubeconfig "$KUBECONFIG_DESTINATION2"
+```
+
+When the above commands return successfully, you can check the peering status by running:
+
+```bash
+kubectl get foreignclusters
+```
+
+The output should look like the following, indicating that an outgoing peering is currently active towards both the *destination1* and the *destination2* clusters, as well as the cross-cluster network tunnels have been established:
+
+```text
+NAME           TYPE     OUTGOING PEERING   INCOMING PEERING   NETWORKING    AUTHENTICATION   AGE
+destination1   InBand   Established        None               Established   Established      41s
+destination2   InBand   Established        None               Established   Established      7s
+```
+
+Additionally, you should have two new virtual nodes in the *origin* cluster, characterized by the install-time provided labels:
+
+```bash
+kubectl get node --selector=liqo.io/type=virtual-node --show-labels
+```
+
+```text
+NAME            STATUS   ROLES   AGE     VERSION   LABELS
+NAME                STATUS   ROLES   AGE   VERSION   LABELS
+liqo-destination1   Ready    agent   57s   v1.25.0   liqo.io/remote-cluster-id=df0baf71-c428-4d2e-8c97-3559e205d34a,liqo.io/type=virtual-node,topology.liqo.io/type=destination
+liqo-destination2   Ready    agent   43s   v1.25.0   liqo.io/remote-cluster-id=3c4604aa-b9b4-4442-be6d-8ffc713139fc,liqo.io/type=virtual-node,topology.liqo.io/type=destination
+```
+
+```{admonition} Note
+Some of the default labels were omitted for the sake of clarity.
+```
+
+## Tune namespace offloading
+
+Now, let's suppose you want to deploy an application that needs to be scheduled on every *destination* cluster, but not in the *origin* one.
+
+```bash
+kubectl create namespace liqo-demo
+```
+
+Then, enable Liqo offloading for that namespace:
+
+```bash
+liqoctl offload namespace liqo-demo \
+  --namespace-mapping-strategy EnforceSameName \
+  --pod-offloading-strategy Remote \
+  --selector 'topology.liqo.io/type=destination'
+```
+
+The above command configures the following aspects (see the dedicated [usage page](/usage/namespace-offloading.md) for additional information concerning namespace offloading configurations):
+
+* the `liqo-demo` namespace, and the contained resources, are offloaded only to the clusters with the `topology.liqo.io/type=destination` label.
+* the pods living in the `liqo-demo` namespace only on virtual nodes.
+
+The *NamespaceOffloading* resource created by *liqoctl* in the `liqo-demo` namespace exposes the status of the offloading process, including a global *OffloadingPhase*, which is expected to be `Ready`, and a list of *RemoteNamespaceConditions*, one for each remote cluster.
+
+In this case both the *destination1* and *destination2* clusters have been selected to offload the namespace `liqo-demo`.
+
+```bash
+kubectl get namespaceoffloadings offloading -n liqo-demo -o yaml
+```
+
```

Comment:
I would add something which explains what is the previous command about, and that the next block of text represents an example of a possible output.

Reply:
This part has been removed and a link to another examples which explains this feature has been added (to avoid duplicated information).